tests/e2e/test_rosacli_operator_roles.go

```diff
@@ -105,6 +105,7 @@ var _ = Describe("Edit operator roles", labels.Feature.OperatorRoles, func() {
 				output, err = ocmResourceService.DeleteOIDCConfig(
 					"--oidc-config-id", oidcConfigID,
 					"--mode", "auto",
+					"--region", profile.Region,
```

Comment:
Although we always provision a cluster using the test profile without overriding its settings, the profile is a template but not an actual configuration for a cluster. In this case, if I specify `Region` from the Prow CI job level, it will cause potential failure.  We should record the cluster configuration in `cluster-config` and use the file as a reference.